sprout.yaml

```diff
@@ -178,13 +178,20 @@ resources:
             exec > >(tee -a /var/log/clearwater-heat-sprout.log) 2>&1
             set -x
 
-            # Set up the signaling network namespace
+            # Set up the signaling network namespace on each boot.
+            cat >/etc/rc.local <<EOF
+            #!/bin/bash
+            set -x
             ip netns add signaling
```

Comment:
Worth a comment as to what we're doing here?  (We have a comment in the parent script, but not in the script we're creating.)